swatch-billable-usage/src/main/java/com/redhat/swatch/billable/usage/services/BillableUsageService.java

```diff
@@ -151,6 +162,7 @@ public BillableUsage produceMonthlyBillable(BillableUsage usage) {
   protected double getTotalRemitted(BillableUsage billableUsage) {
     var filter = BillableUsageRemittanceFilter.fromUsage(billableUsage);
     filter.setExcludeFailures(true);
+    filter.setExcludeGratisStatus(true);
```

Comment:
I got confused because we're using this amount in the method `calculateBillableUsage` to calculate the actual usage calculation... But I'm reverting this change.